pkg/clusterd/disk.go

```diff
@@ -71,6 +69,10 @@ func DiscoverDevices(executor exec.Executor) ([]*sys.LocalDisk, error) {
 			continue
 		}
 
+		if isFilter && !deviceMatchWithFilter(d, deviceName) {
+			continue
```

Comment:
How about logging that the device doesn't match the filter?

Reply:
Completely agree, I've added logger